src/Riok.Mapperly/Emit/Syntax/SyntaxFactoryHelper.cs

```diff
@@ -64,6 +64,22 @@ public static SeparatedSyntaxList<T> CommaSeparatedList<T>(params T[] nodes)
         return SeparatedList<T>(joinedNodes);
     }
 
+    public SeparatedSyntaxList<T> ConditionalCommaLineFeedSeparatedList<T>(IEnumerable<T> nodes)
+        where T : SyntaxNode
+    {
+        var nodesList = nodes.ToList();
+        var sep = Token(SyntaxKind.CommaToken);
+        if (nodesList.Count > 1)
```

Comment:
Consider extracting the threshold in a named constant to improve the readability.